src/renderer/ScreenController.js

```diff
@@ -59,11 +59,8 @@ class Home extends React.Component {
       case 'CreateGroup':
         Screen = CreateGroup
         break
-      case 'ChatView':
-        Screen = ChatView
-        break
       default:
-        Screen = ChatList
+        Screen = SplittedChatListAndView
```

Comment:
Maybe just keep the name as `ChatView` and throw everything else away?

Reply:
I don't have any hard opinions on naming, but maybe it makes more sense to use new new names for the components. I will think about it, maybe i have an idea.